docker/micro-cluster-lab/mysql/init/test_create.sql

```diff
@@ -0,0 +1,3 @@
+Create table test(id INT NOT NULL, value INT NOT NULL, data DATE NOT NULL);
```

Comment:
Change **Create table** to **CREATE TABLE**

Reply:
ok